core/src/main/java/com/vityuk/ginger/provider/LocalizationProvider.java

```diff
@@ -41,7 +42,13 @@
 
     String getMessage(String key, Object... parameters);
 
+    MessageFormat getMessageFormat(String key);
+
     String getSelectedMessage(String key, String selector, Object... parameters);
 
-    String getPluralMessage(String key, int count, Object... parameters);
+    MessageFormat getSelectedMessageFormat(String key, String selector);
+
+    String getPluralMessage(String key, Number count, Object... parameters);
```

Comment:
The problem with replacing int with Number is that in some languages form for double is different from int. Like "3 tableS" is not the same as "3.5 tableS".